src/GEOUNED/__init__.py

```diff
@@ -272,6 +272,7 @@ def Start(self):
        if not Options.Facets:
 
           # decompose all solids in elementary solids (convex ones)
+          print(len(MetaList))
```

Comment:
```suggestion
```

this looks a bit like a debug print statement, was this left accidently or is it good to print here?